src/DataFixtures/SongFixtures.php

```diff
@@ -24,6 +24,17 @@ public function load(ObjectManager $manager): void
         $manager->persist($song);
         $manager->flush();
 
-        $this->addReference('firstSong', $song);
+
+        $artist = new Artist('Up and Coming Artist');
```

Comment:
Shouldn't we create artist and album in their respective fixtures files and reference them here ?

Reply:
Nevermind, i saw you used the fixture files in another way (per test cases instead of per entity)

Reply:
😊